Add additional documentation for setting an owner on contract deployment (#149)
FuelLabs/sway-libs: libs/ownership/README.md

The Ownership library provides a way to block users other than a single "owner" or "admin" from calling functions. Ownership is often used when needing administrative calls on a contract.

For more information please see the [specification](./SPECIFICATION.md).

# Using the Library

## Getting Started

In order to use the Ownership library it must be added to the Forc.toml file and then imported into your Sway project. To add Sway-libs as a dependency to the Forc.toml file in your project please see the [README.md](../../README.md).

You may import the Ownership library's functionalities like so:

```rust
use ownership::Ownership;
```

Once imported, the `Ownership` struct should be added to the storage block of your contract. There are two approaches when declaring ownership in storage.

> 1. Initalize the owner on contract deployment by calling the `initalized()` function.

Reviewer: ```suggestion
1. Initialize the owner on contract deployment by calling the `initialized()` function.
```

```rust
storage {

>     owner: Ownership = Ownership::initalized(Identity::Address(Address::from(0x0000000000000000000000000000000000000000000000000000000000000000))),

Reviewer: ```suggestion
    owner: Ownership = Ownership::initialized(Identity::Address(Address::from(0x0000000000000000000000000000000000000000000000000000000000000000))),
```

}
```

> 2. Leave the owner uninitialized and call the `set_ownership()` function in your own constructor.

Reviewer: ```suggestion
2. Leave the owner uninitialized and call the `set_ownership()` function in your own constructor.
```

Author: There is nothing different here

Reviewer: Yh, I resolved it for that reason. I refreshed partly through review when github had a popup button so maybe something changed to be the same
